portal-web/test/functional/com/liferay/portalweb/tests/development/tools/testinginfrastructure/block/macro/TestUtils.macro

```diff
@@ -37,6 +37,19 @@ definition {
 		}
 	}
 
+	macro assertPartialEquals {
```

Comment:
don't we have something global to assert string partially? 🤔

Reply:
I think we may need to assert some partial text in response for all headless tests. For example, this file is for StructuredContent, and there is one more ticket for Document. What do you think of this?

Reply:
yeah, cool! I've checked and the other partial function has selenium and element attribute involved so that's a proper method and place to create this one :) Thanks!